module/Activity/view/activity/activity/view.phtml

```diff
@@ -144,4 +144,9 @@ $this->headTitle($this->translate('Activities')); ?>
         echo $this->formSubmit($submit);
         echo $this->form()->closeTag();?>
     <?php endif; ?>
+
+    <?php if ($this->acl('activity_acl')->isAllowed('activity', 'create')): ?>
```

Comment:
This doesn't quite seem right. Since not everyone can export this list for all activities. I think it would be better to add a page to the admin interface which lists all activities which someone has access to. Adding such a page also is handy if we implement things such as editing later.